python/cuml/fil/fil.pyx

```diff
@@ -181,6 +187,18 @@ cdef class ForestInference_impl():
                             ' to the documentation')
         return algo_dict[algo_str]
 
+    def get_storage_type(self, storage_type_str):
+        storage_type_dict={'AUTO': storage_type_t.AUTO,
```

Comment:
Would be shorter just to have lowercase keys and do:
`storage_type_str.lower() not in storage_type_dict.keys()`
Sorry my original note on that was confusing!

Reply:
Unicode uppercase/lowercase conversion has a lot of corner cases, which I'd like to avoid. Therefore, I'll keep the two-key solution for now (one for lowercase, one for uppercase).